tests/test_utils.py

```diff
@@ -3491,6 +3491,51 @@ def test_load_x963_vectors():
     ]
 
 
+def test_load_kbkdf_vectors():
```

Comment:
I hate to ask but would you mind pulling this test (and the loader in utils) into a separate PR so we can review/land it separately?